manifests/m/MOTU/MSeries/4.0.88030/MOTU.MSeries.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: MOTU.MSeries
+PackageVersion: 4.0.88030
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://motu.com/en-us/download-center/download/42
+  InstallerSha256: CAF880E121BE3572FDAA49BA320031D414B95F1D8508CAA4B974C7C54D051EAD
+  ProductCode: com_motu_installer_core_uac_is1
+  InstallerSwitches:
+    Silent: /SILENT /NORESTART
+    SilentWithProgress: /SILENT /NORESTART

```

```suggestion
  InstallerSwitches:
    Custom: /NORESTART
```
You can do this instead, WinGet CLI will add /VERYSILENT and /SILENT depending on the switch.